src/api/cart/entity.ts

```diff
@@ -36,3 +36,16 @@ export interface CartSummaryResponse {
   cart_items_amount: number;
   is_available: boolean;
 }
+
+export interface AddCartRequest {
+  menuInfo: {
+    orderable_shop_id: number;
+    orderable_shop_menu_id: number;
+    orderable_shop_menu_price_id: number;
+    orderable_shop_menu_option_ids: {
+      option_group_id: number;
+      option_id: number;
+    }[];
+    quantity: number;
+  };
+}
```

Comment:
저는 코드의 depth가 줄어들어서 보기 편하다고 생각해서 `MenuInfo`(혹은 다른 타입명)라는 인터페이스를 따로 만들어서 지정할 거 같습니다! (개인적 의견입니다. 참고만 해주세요!)
```suggestion
interface MenuOptionId {
  option_group_id: number;
  option_id: number;
}

export interface MenuInfo {
  orderable_shop_id: number;
  orderable_shop_menu_id: number;
  orderable_shop_menu_price_id: number;
  orderable_shop_menu_option_ids: MenuOptionId[];
}

export interface AddCartRequest {
  menuInfo: MenuInfo[];
  quantity: number;
};
```